workflows/sarek.nf

```diff
@@ -212,73 +212,73 @@ include { PREPARE_INTERVALS                                    } from '../subwor
 include { PREPARE_CNVKIT_REFERENCE                             } from '../subworkflows/local/prepare_cnvkit_reference'
 
 // Convert BAM files to FASTQ files
-include { ALIGNMENT_TO_FASTQ as ALIGNMENT_TO_FASTQ_INPUT       } from '../subworkflows/nf-core/alignment_to_fastq'
-include { ALIGNMENT_TO_FASTQ as ALIGNMENT_TO_FASTQ_UMI         } from '../subworkflows/nf-core/alignment_to_fastq'
+include { ALIGNMENT_TO_FASTQ as ALIGNMENT_TO_FASTQ_INPUT         } from '../subworkflows/nf-core/alignment_to_fastq'
+include { ALIGNMENT_TO_FASTQ as ALIGNMENT_TO_FASTQ_UMI           } from '../subworkflows/nf-core/alignment_to_fastq'
 
 // Run FASTQC
-include { RUN_FASTQC                                           } from '../subworkflows/nf-core/run_fastqc'
+include { RUN_FASTQC                                             } from '../subworkflows/nf-core/run_fastqc'
 
 // TRIM/SPLIT FASTQ Files
-include { FASTP                                                } from '../modules/nf-core/modules/fastp/main'
+include { FASTP                                                  } from '../modules/nf-core/modules/fastp/main'
 
 // Create umi consensus bams from fastq
-include { CREATE_UMI_CONSENSUS                                 } from '../subworkflows/nf-core/fgbio_create_umi_consensus/main'
+include { CREATE_UMI_CONSENSUS                                   } from '../subworkflows/nf-core/fgbio_create_umi_consensus/main'
 
 // Map input reads to reference genome
-include { GATK4_MAPPING                                        } from '../subworkflows/nf-core/gatk4/mapping/main'
+include { GATK4_MAPPING                                          } from '../subworkflows/nf-core/gatk4/mapping/main'
 
 // Merge and index BAM files (optional)
-include { MERGE_INDEX_BAM                                      } from '../subworkflows/nf-core/merge_index_bam'
+include { MERGE_INDEX_BAM                                        } from '../subworkflows/nf-core/merge_index_bam'
 
-include { SAMTOOLS_CONVERT as SAMTOOLS_CRAMTOBAM               } from '../modules/nf-core/modules/samtools/convert/main'
-include { SAMTOOLS_CONVERT as SAMTOOLS_CRAMTOBAM_RECAL         } from '../modules/nf-core/modules/samtools/convert/main'
+include { SAMTOOLS_CONVERT as SAMTOOLS_CRAMTOBAM_MARKDUPLICATES  } from '../modules/nf-core/modules/samtools/convert/main'
```

Comment:
why not
`CRAMTOBAM_MARKDUPLICATES`, `CRAMTOBAM_RECAL` and `CRAMTOBAM_VARIANTCALLING`?

Reply:
or do we need the `SAMTOOLS` prefix in `conf/modules.config`?